src/Core/DevOps/System/Command/SystemDumpDatabaseCommand.php

```diff
@@ -22,6 +23,13 @@ public function __construct(string $defaultDirectory, Connection $connection)
         $this->connection = $connection;
     }
 
+    protected function configure()
+    {
+        parent::configure();
```

Comment:
You can safely remove this call, because the parent function is empty :)

Reply:
I have a different opinion on "safely remove" but I removed it